POEApi.Model/Item.cs

```diff
@@ -111,8 +115,11 @@ protected Item(JSONProxy.Item item)
 
             TradeX = X;
             TradeY = Y;
-            TradeInventoryId =InventoryId;
+            TradeInventoryId = InventoryId;
             Character = string.Empty;
+
+            if (item.Elder || item.Shaper)
+                BackgroundUrl = ItemBackgroundUrlBuilder.GetUrl(this);
```

Comment:
I'm not personally a massive fan of skipping braces on single if statements, you're the owner though so it's entirely up to you.

Reply:
We have both styles all over Procurement, I should probably decide on one and update it across the code base as I did with the renaming.